src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsTerminal/Properties/Resources.resx

```diff
@@ -120,6 +120,12 @@
   <data name="open_new_tab" xml:space="preserve">
     <value>Open profiles in a new tab</value>
   </data>
+  <data name="open_quake" xml:space="preserve">
+    <value>Open profiles in the quake window</value>
+  </data>
```

Comment:
`<comment>Quake is a well-known computer game. Don't localize. See https://en.wikipedia.org/wiki/Quake_(video_game)</comment>`
Add a comment for localization.